contracts/src/oracle_implementation/OracleImplementation.cairo

```diff
@@ -4,6 +4,7 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.bool import TRUE, FALSE
 
 from entry.structs import Entry
+from proxy.library import Proxy
```

Comment:
Why is this in oracle implementation? Think we should fold it all into OracleController which should just be called Oracle

Reply:
Will do in a separate PR